test/views/cls.njk

```diff
@@ -34,19 +34,41 @@
   <script type="module">
     const {onCLS} = await __testImport('{{ modulePath }}');
 
+    const queue = new Set();
+    function addToQueue(metric) {
+      queue.add(metric);
+    }
+
     onCLS((cls) => {
       cls.instance = 1;
 
       // Log for easier manual testing.
       console.log('CLS:', cls);
 
-      // Test sending the metric to an analytics endpoint.
-      navigator.sendBeacon(`/collect`, JSON.stringify(__toSafeObject(cls)));
+      if (self.__documentVisibilityChangeHandling) {
```

Comment:
I would call this something different (perhaps `self.__batchReporting`) because right now it's not very clear what this does.

Reply:
Fixed